android/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">Jitsi Meet</string>
+    <string name="restriction_server_url_description">URL of Jitsi Videoserver instance to connect</string>
```

Comment:
It will be shown by MDM/EMM UI.
For example, MDM can integrate an iFrame from Google which is fully generated from this XML.
(https://developers.google.com/android/work/play/emm-api/managed-configurations-iframe)